src/listmode_utilities/lm_fansums.cxx

```diff
@@ -102,6 +102,7 @@ set_defaults()
   store_prompts = true;
   delayed_increment = -1;
   interactive=false;
+  current_frame=0;
```

Comment:
this actually seems a variable that's never used. let's remove it instead.

Reply:
With this one I agree, I will make it disappear soon.